### ISSUE TITLE
Active tab is too similar to non-active, ubuntu unity 16.04 

### ISSUE THREAD

**Issue by Strate** [gnunn1/tilix#1523]
Hi, thank you for this awesome project.

I found it has an option to show sessions as tab. But, tabs looks like too "flat", active tab can be easy mixed with non-active, see screenshot:

![image](https://user-images.githubusercontent.com/392644/45641334-db8f7f80-babd-11e8-9d7a-211e7de01d42.png)

Right now right-most tab is active. Could you please suggest how can it be customized?

**Comment by gnunn1** [comment]
This is controlled by the GTK theme and not Tilix and you should see the same behavior in other applications like gedit (unless Ubuntu is patching specific apps for this). If you are using the default Ubuntu theme please open an issue with Ubuntu, if it is a third party theme then open an issue against that theme.

**Comment by Strate** [comment]
Thank you for advice. I found the solution, simply put to your `~/.config/gtk-3.0/gtk.css` file:

```css
@define-color terminal_bg #300a24;

.tilix-background.notebook.header {
    background-color: @dark_bg_color;
    border: none;
}


.tilix-background.notebook tab.top {
    background-color: shade(@bg_color, 0.7);
    border-width: 1px 1px 0px 1px;
    border-radius:  8px 8px 0px 0px;
    border-image: none;
    border-style: solid;
    border-color: @dark_bg_color;
}

.tilix-background.notebook tab.top:active {
    background-color: @bg_color;
    border-image: none;
}

.tilix-background.notebook tab .button {
    background-color: transparent;
    padding: 1px;
}

/* Match the actions bar.  This is a hack and should be fixed in 3.20 */
.tilix-background.notebook GtkBox.vertical {
    background-color: transparent;
}

.background .tilix-background.notebook GtkBox.vertical .button {
    border-image:none;
    border-color: transparent;
    border-width: 1px;
    border-radius: 5px;
    padding: 1px;
}
```

**Comment by xird** [comment]
The above CSS didn't work for me (Tilix 1.7.7 from Ubuntu), and I couldn't find a way to highlight the active tab. This changes the background for the inactive tabs, which is good enough for me.

tab {
    background-color: #888888;
}
